src/target/riscv/riscv.c

```diff
@@ -4795,7 +4795,7 @@ int riscv_init_registers(struct target *target)
 
 		} else if (number >= GDB_REGNO_V0 && number <= GDB_REGNO_V31) {
 			r->caller_save = false;
-			r->exist = riscv_supports_extension(target, 'V') && info->vlenb;
+			r->exist = info->vlenb && info->vlenb;
```

Comment:
Only one of them is needed:
`r->exist = info->vlenb;`

Reply:
Oops, too quick with the search and replace.